DuckDB.NET.Data/DuckDBConnection.TableFunction.cs

```diff
@@ -19,6 +19,12 @@ public record TableFunction(IReadOnlyList<ColumnInfo> Columns, IEnumerable Data)
 partial class DuckDBConnection
 {
 #if NET8_0_OR_GREATER
+    [Experimental("DuckDBNET001")]
+    public void RegisterTableFunction(string name, Func<IReadOnlyList<IDuckDBValueReader>, TableFunction> resultCallback, Action<object?, IDuckDBDataWriter[], ulong> mapperCallback)
```

Comment:
I think in this case we can remove the parameter that the Func receives.

Reply:
Removed.